esmvaltool/recipes/recipe_multimodel_products_wp5.yml

```diff
@@ -0,0 +1,80 @@
+###############################################################################
+# nameslist_multimodel_products.yml
+#
+# Description
+# Namelist to compute the ensemble mean anomaly and the ensemble
+# variance and agreement and plot the results as maps and time series.
+#
+# Authors
+# Nicolaur Manubens (BSC)
+# Alasdair Hunter (BSC)
+#
+#
+# Project
+# KNMI Magic
+#
+# Reference
+# Manubens, Nicolau, Louis-Philippe Caron, Alasdair Hunter, Omar Bellprat,
+# Eleftheria Exarchou, Neven S. Fučkar, Javier Garcia-Serrano et al.
+# "An R package for climate forecast verification." Environmental
+# Modelling & Software 103 (2018): 29-42.
+#
+#
+# This namelist is part of the ESMValTool
+###############################################################################
+---
+
+datasets:
+  - {dataset: bcc-csm1-1, type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 1961,  end_year: 1990}
+#  - {dataset: GFDL-ESM2G, type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 2000,  end_year: 2002}
+  - {dataset: MPI-ESM-MR, type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 1961,  end_year: 1990}
+  - {dataset: MIROC5,     type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 1961,  end_year: 1990}
+  - {dataset: MPI-ESM-MR, type: exp, project: CMIP5,  mip: Amon,  exp: rcp26,  ensemble: r1i1p1,  start_year: 2006,  end_year: 2099}
+  - {dataset: bcc-csm1-1, type: exp, project: CMIP5,  mip: Amon,  exp: rcp26,  ensemble: r1i1p1,  start_year: 2006,  end_year: 2099}
+  - {dataset: MIROC5,     type: exp, project: CMIP5,  mip: Amon,  exp: rcp26,  ensemble: r1i1p1,  start_year: 2006,  end_year: 2099}
+
+#latlon_roi: (/latmin,latmax,lonmin,lonmax/)
+preprocessors:
+  preproc:
+    regrid:
+      target_grid: bcc-csm1-1
+      scheme: linear
+    mask_fillvalues:
+      threshold_fraction: 0.95
+    extract_region:
+      start_longitude: 0
+      end_longitude: 360
+      start_latitude: -90
+      end_latitude: 90
+
+diagnostics:
+  anomaly_agreement:
+    description: Calculate multi-member anomalies and their spread/agreement.
+    variables:
+      tas:
+        preprocessor: preproc
+        mip: Amon
+        field: T2M
+    scripts:
+      main:
+        script: magic_bsc/multimodel_products.r #(in diag_scripts)
+
+        #Specify datasets to be used for computing climatology and anomaly
+        climatology_class: historical
+        anomaly_class: rcp26
+
+        climatology_start_year: 1961
```

Comment:
It looks like these items (climatology class, start_year, end_year, anomaly class, start year, end year) provide the same information that is already in the preprocessor again. It might be more convenient if you could read it from there instead of having to specify it two times in the recipe. Actually, I think you could write this recipe a bit nicer by making groups of datasets like so:
```yaml
datasets:
  - {dataset: bcc-csm1-1}
  - {dataset: MPI-ESM-MR}
  - {dataset: MIROC5}

...

diagnostics:
  anomaly_agreement:
    description: Calculate multi-member anomalies and their spread/agreement.
    variables:
      climatology: &variable
        short_name: ta
        preprocessor: preproc
        mip: Amon
        field: T2M
        start_year: 1961
        end_year: 1990
        type: exp
        project: CMIP5
        mip: Amon,
        exp: historical
        ensemble: r1i1p1
      anomaly:
        <<: *variable
        exp: rcp26
        start_year: 2006
        end_year: 2099


```